docs/application/dotnet/guides/connectivity/stc.md

```diff
@@ -0,0 +1,166 @@
+# STC
+
+
+STC means Smart Traffic Control. It provides extensible packet-level control services, including per-app data usage, total data quota, and background app's data saving.
+
+The main features of the Tizen.Network.Stc namespace include:
+
+- Retrieve Data Usage For System
+
+  You can [retrieve network data consumed by system](#retrieve-data-usage-for-system).
```

Comment:
Please make this small change:

You can [retrieve the network data consumed by the system](#retrieve-data-usage-for-system).